repose-aggregator/artifacts/valve/src/test/scala/org/openrepose/valve/ReposeJettyServerTest.scala

```diff
@@ -24,14 +24,14 @@ import org.junit.runner.RunWith
 import org.openrepose.core.container.config.SslConfiguration
 import org.openrepose.core.spring.{CoreSpringProvider, ReposeSpringProperties}
 import org.scalatest.junit.JUnitRunner
-import org.scalatest.{BeforeAndAfterAll, FunSpec, Matchers}
+import org.scalatest.{FunSpec, Matchers}
 
 @RunWith(classOf[JUnitRunner])
-class ReposeJettyServerTest extends FunSpec with Matchers with BeforeAndAfterAll {
+class ReposeJettyServerTest extends FunSpec with Matchers {
 
-  override def beforeAll() = {
-    CoreSpringProvider.getInstance().reInitializeCoreContext("/config/root", false)
-  }
+  CoreSpringProvider.getInstance().reInitializeCoreContext("/config/root", false)
```

Comment:
Follow-up. I removed it to drop a trait because I didn't think it was necessary.

I could have been wrong, though. In which case I'll change this back.